tests/api/route-services/projects.test.js

```diff
@@ -404,4 +405,33 @@ describe('tests for the projects service', () => {
       })
       .then(() => done());
   });
+
+  test('createProject creates entries in projects and samples correctly', (done) => {
+    const updateItemSpy = mockDynamoUpdateItem({});
+
+    const expectedSamplesUpdate = {
+      TableName: 'samples-test',
+      Key: AWS.DynamoDB.Converter.marshall({ experimentId: 'experiment-1' }),
+      UpdateExpression: 'SET samples = :emptySamples, projectUuid = :projectUuid',
+      ExpressionAttributeValues: AWS.DynamoDB.Converter.marshall({ ':emptySamples': {}, ':projectUuid': 'project-1' }),
+      ReturnValues: 'UPDATED_NEW',
+    };
+
+    const expectedProjectsUpdate = {
+      TableName: 'projects-test',
+      Key: AWS.DynamoDB.Converter.marshall({ projectUuid: 'project-1' }),
+      UpdateExpression: 'SET projects = :project',
+      ExpressionAttributeValues: AWS.DynamoDB.Converter.marshall({ ':project': mockProject }),
+      ReturnValues: 'UPDATED_NEW',
+    };
+
+    (new ProjectsService()).createProject('project-1', mockProject)
+      .then((res) => {
+        expect(res).toEqual(OK());
+
+        expect(updateItemSpy.mock.calls).toContainEqual([expectedSamplesUpdate]);
+        expect(updateItemSpy.mock.calls).toContainEqual([expectedProjectsUpdate]);
+      })
+      .then(() => done());
```

Comment:
Rather than doing this, can you make this test to be `async`, and then break this piece of code into `awaits`? I think it' easier to read,

Reply:
Ok, should I do this with all the other tests? Since they all have this pattern